apps/kitchen-sink/src/ensemble/screens/widgets.yaml

```diff
@@ -20,6 +20,29 @@ View:
             text: Widgets
         - Markdown:
             text: More to come! In the meantime, checkout the Ensemble [documentation](https://docs.ensembleui.com/).
+
+        - Markdown:
+            text: "## Collapse"
+        - Collapsible:
+            id: collapseWidget
+            value:
+              - "1"
+            expandIconPosition: "end"
+            items:
```

Comment:
This should take a generic list of children like Column/Row

Reply:
@evshi I think here we do not need of passing children like column/rows,
because here we used prebuilt collapsible widget of antd design.

here is the reference,
https://ant.design/components/collapse